src/app/portlets/dot-templates/dot-template-advanced/dot-template-advanced.component.html

```diff
@@ -0,0 +1,57 @@
+<dot-portlet-base>
+    <dot-portlet-toolbar [actions]="actions"> </dot-portlet-toolbar>
+
+    <div class="content-wrapper">
+        <form class="p-fluid" [formGroup]="group">
+            <div class="p-field">
+                <label for="title">Title</label>
```

Comment:
We need to add a `*` to the required field. We normally just add it to the DOM but I was thinking create a class and add it at the `:after` selector so if we have to change it in all the required fields will be only a class. Thoughts?